### ISSUE TITLE
Nullpointer when teleporting to zone 1

### ISSUE THREAD

**Issue by HgeX** [espidev/ProtectionStones#8]
This error happens sometimes, when a player tries to teleport to home 1. The others (I mean home 2, 3, etc) works just fine, but 1 doesn't for some reason. Also interesting that this applies for zones, that were created before I updated the plugin on my server. The old plugin's version was 1.7.1, the new plugin's version is 2.0.4.

```> [08:08:20 WARN]: [ProtectionStones] Plugin ProtectionStones v2.0.4 generated an exception while executing task 5314
java.lang.NullPointerException: null
at me.vik1395.ProtectionStones.commands.ArgTp.lambda$argumentTp$3(ArgTp.java:143) ~[?:?]
at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:82) ~[patched_1.13.2.jar:git-Paper-523]
at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:52) ~[patched_1.13.2.jar:git-Paper-523]
at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[patched_1.13.2.jar:git-Paper-523]
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) [?:1.8.0_212]
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) [?:1.8.0_212]
at java.lang.Thread.run(Thread.java:748) [?:1.8.0_212]
```

**Comment by espidev** [comment]
Hmm... It seems that the region didn't get upgraded in the upgrade process. What is the id of the region?

**Comment by HgeX** [comment]
```ps4214x62y9547z:
        min: {x: 4164, y: -194, z: 9497}
        max: {x: 4264, y: 318, z: 9597}
        members:
            unique-ids: [ad169060-b4fb-3d9a-99b7-15c9451203d6, 6b0fb79b-4dd0-341c-a655-5b77d4d7460c]
        flags: {greeting: §2❯❯ §7Beléptél §aMMpro §7területére..., farewell: §4❯❯
                §7Elhagytad §cMMpro §7területét...}
        owners:
            unique-ids: [dc23e1b1-945f-324b-9df7-3a5d40c2a98a]
        type: cuboid
        priority: 0
```

This is one of the regions, but many (if not all) of them are affected.

**Comment by espidev** [comment]
try running ps admin fixregions when your server isn't too busy (it lags)

**Comment by HgeX** [comment]
Thanks, I'll try that today evening.

**Comment by HgeX** [comment]
Unfortunately using `/ps admin fixregions` didn't solve the issue. The command finished almost immediately, but nothing changed. Some regions have names at their owner and member section, while others have uuid-s.

**Comment by espidev** [comment]
Hmmmm, do you see a flag called PS_HOME and PS_MATERIAL set in the affected regions?

**Comment by HgeX** [comment]
Yes, both flags are set on all of the regions, but now I see, that on some of them the ps block material is wrong. I mean, on one of them it says `light_gray_stained_glass`, but we never had that block as a protectionstone.

**Comment by espidev** [comment]
Ah, I see, so the upgrade process messed up. If it's not too much of a bother you could simply set the material manually, and hopefully everything would be fixed.

**Comment by HgeX** [comment]
Yeah, it seems like. I'll do that later today, and let you know the result! :D

**Comment by espidev** [comment]
If there are still issues I can come on your server to check it out

**Comment by HgeX** [comment]
Okay, it seems, that the issue is solved now. I'm not 100% sure about, what happend, but here is my theory: when I switched from 1.7.1 to 2.0.x (I don't remember the exact version), the hiddenstones.yml got ignored for some reason. Then, when I used the `ps admin fixregions`command, it took the block in the center of every zone, and put it's material into the region flags. Anyways, the issue is now fixed, thank you really much for your support!
